03_Operators/7_Logical_Assignment_Operators/README.md

```diff
@@ -0,0 +1,8 @@
+<h3>JavaScript Logical Assignment Operators:</h3>
+
+- logical OR assignment (`||=`):
+    * assigns the right operand to the left operand if the left operand is falsy.
+- logical AND assignment (`&&=`)
+    * assigns the right operand to the left operand if the left operand is truthy.
+- Nullish coalescing assignment (`&&=`)
```

Comment:
The Nullish coalescing operator is `??`. So,
`&&=` -> `??=`

Reply:
fixed.